Lib/asyncio/locks.py

```diff
@@ -168,6 +168,12 @@ def __init__(self, *, loop=None):
                           "and scheduled for removal in Python 3.10.",
                           DeprecationWarning, stacklevel=2)
 
+        if not self._loop.is_running():
+            warnings.warn("The asyncio objects created without running "
+                          "event loop is deprecated since Python 3.9 and "
+                          "scheduled for removal in Python 3.10.",
+                          DeprecationWarning, stacklevel=2)
```

Comment:
The implementation of the warning looks fine, but the message itself could use some grammatical fixes.

Also, @1st1 @asvetlov, what do you think about the 3.10 removal version? IMO, this removal would end up breaking far too much code to be changed in the next version, and even two versions might be too early still. I think for now we should leave it as a `DeprecationWarning`, and remove it later down the road when users have had plenty of time to adjust rather than having a fixed removal date. Especially considering that we'll be breaking a substantial amount of code already with the event loop argument removals.

With the above in mind, I'd suggest changing the warning message to the following:
```
The creation of asyncio objects without a running
event loop is deprecated as of Python 3.9.
```
(This also applies to the warning message for `asyncio.Queue`)